File: finah-web/Finah_Web/Models/SuperKlasseAanvraagAccount.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Finah_Web.Models
{
    public abstract class SuperKlasseAanvraagAccount
    {

        #region Public members
        public int Id { get; set; }

        public string Naam { get; set; }

        public string VoorNaam { get; set; }

        public string RijksRegisterNr { get; set; }

        public string Adres { get; set; }

        public Postcode Postcd { get; set; }

        public string Telnr { get; set; }

        public string Gsm { get; set; }

        public string Login { get; set; }

        public string Passwd { get; set; }

        public string GeheimeVraag { get; set; }

        public string GeheimAntwoord { get; set; }
        public TypeAccount TypeAcc { get; set; }
        #endregion
    }


    #region Enum
    public enum TypeAccount
    {
        Hulpverlener,
        Onderzoeker
    }
    #endregion
}

File: finah-web/PHP/Models/SuperKlasseAanvraagAccount.php
<?php

    /**
     * Created by PhpStorm.
     * User: Gert
     * Date: 25/03/2015
     * Time: 23:18
     */
    class SuperKlasseAanvraagAccount
    {
        public $Id;
        public $Naam;
        public $Voornaam;
        public $Adres;
        public $Postcd;
        public $Telnr;
        public $Login;
        public $Passwd;
        public $Email;
        public $TypeAcc;

        function __construct($Id = null, $Naam = null, $Voornaam = null, $Adres = null, $Postc=null,$Telnr = null, $Login = null, $Passwd = null, $Email = null, $TypeAccount = null)
        {
            $this->Id = $Id;
            $this->Naam = $Naam;
            $this->Voornaam = $Voornaam;
            $this->Adres = $Adres;
            $this->Postc = $Postc;
            $this->Telnr = $Telnr;
            $this->Login = $Login;
            $this->Passwd = $Passwd;
            $this->Email = $Email;
            $this->TypeAcc = $TypeAccount;
        }


    }